### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by dedribble** [SinanMtl/vue-rate#29] [General]
I am trying to use this plugin in my nuxt app, after creating vue-rate.js and importing the package I was getting an error
"Could not find a declaration file for module 'vue-rate'. 'c:/Users/ADEDAMOLA/Desktop/run-app/node_modules/vue-rate/dist/vue-rate.umd.min.js' implicitly has an 'any' type. Try `npm i --save-dev @types/vue-rate` if it exists or add a new declaration (.d.ts) file containing `declare module 'vue-rate';`" Please I will appreciate any help.

**Comment by SinanMtl** [comment]
Hi @dedribble,
I added type definitions. You must update `vue-rate` to 2.4.1
